pytroll_collectors/segments.py

```diff
@@ -491,6 +535,23 @@ def _find_time_slot(self, time_obj):
 
         return str(time_obj)
 
+    def check_schedule_time(self, check_time, raw_hour, raw_minute):
+        # Check if raw time is inside configured interval
+        toret = False
```

Comment:
It took me some time to understand what `toret` meant, can you rename this to `to_ret` or even something more specific like eg `time_ok` ?